src/Fields/Text.php

```diff
@@ -50,6 +50,9 @@ public function displayInEdit($object, $inline = false)
 
     protected function getFieldType()
     {
+        if ( strpos(strtolower($this->field), 'password') !== false ) {
```

Comment:
es una mica guarro aixo no?

Reply:
mmm es una manera àgil de diferenciar si es una contrasenya crec

Reply:
pero esta molt oculta, si mai vols crear un field que tingui password per el mig, i no faci hidden no ho podras fer

Reply:
Penso que no tindria sentit crear un field que tingui password per el mig i que no fos un password... i per normativa el password ha de anar amagat

Reply:
@BadChoice com ho vols fer sinó, creo un mètode del Field Text, que sigui isPassword() i que faci això?

Reply:
crec que el més sensat es canviar al projecte el tipus a Password on faci falta i descartar aquest check dins de trust

Reply:
Sii, però es per les integracions, on directament se li diu el nom del camp, i sempre fa tipus Text... com ho faries sino?
Exemple:
![Captura de pantalla 2021-04-28 a las 16 54 42](https://user-images.githubusercontent.com/29859922/116425232-6f16b880-a842-11eb-83fc-9379e3e244c7.png)
![Captura de pantalla 2021-04-28 a las 16 55 15](https://user-images.githubusercontent.com/29859922/116425346-848be280-a842-11eb-8b84-cc70bb76247a.png)

Reply:
Veig que hi ha algunes integracions que directament es construeixen els fields (per tant poso el Password:make()) i altres com el q the passat captura que van així, llavors faig el que volia fer en aquesta PR però directament al projecte de retail.
Tanco PR 👌🏽